apprise/plugins/NotifyBase.py

```diff
@@ -288,7 +330,7 @@ def notify(self, body, title=None, notify_type=NotifyType.INFO,
 
             except TypeError:
                 # bad attachments
-                return False
+                raise
```

Comment:
Not sure if we should change how the logic is handled here? thoughts?

Reply:
In general, I would absolutely _love_ to see the code raise more exceptions instead of just returning `False`. But I don't know yet if that resonates well with Chris. Maybe something for a future _major_ release?

Reply:
I enjoy the discussion. We can keep it if it makes sense. I just saw a logic change from where it was. I'll mark this resolved; leave this part as you have it. 🙂

Reply:
You can't `return` in a generator, so the exception bubbles up to the caller, which is either .notify() or .async_notify(). And both have try/except branches that will catch a TypeError and return False. So, in the end, there's no change to the exception handling logic. :+1:

I agree with @amotl that passing through exceptions seems more Pythonic than reporting True/False, but that's a discussion for another day.

Reply:
You guys sold me; no worries here. marking issue resolved.